CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.1)
 project(libuvc
   VERSION 0.0.7
-  LANGUAGES C
+  LANGUAGES C CXX
```

Comment:
maybe best to `enable_language(CXX)` under `if(BUILD_TEST)`

Reply:
Yes, that's much better